src/backend/aspen/api/views/pango_lineages.py

```diff
@@ -0,0 +1,24 @@
+import sqlalchemy as sa
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from aspen.api.deps import get_db
+from aspen.api.schemas.pango_lineages import PangoLineagesResponse
+from aspen.database.models import PangoLineage
+
+router = APIRouter()
+
+
+@router.get("/", response_model=PangoLineagesResponse)
+async def list_pango_lineages(db: AsyncSession = Depends(get_db)):
+    """Gets all the Pango lineages.
+
+    Note that the returned result is very simple: pretty  much just a list of
+    strings. While we could return `id` information, the ids (at least with the
+    current data workflow) are ephemeral and have no connection to the "real"
+    lineage info. The only real info we have is the names of the lineages,
+    so that's all we pull and return.
+    """
+    all_lineages_query = sa.select(PangoLineage.lineage)  # type: ignore
+    result = await db.execute(all_lineages_query)
+    return {"all_pango_lineages": result.scalars().all()}
```

Comment:
This should return a response schema rather than a raw object -- see https://github.com/chanzuckerberg/czgenepi/blob/trunk/src/backend/aspen/api/views/locations.py#L30

Reply:
I thought that was accomplished via the `response_model=PangoLineagesResponse` in the decorator? When I was working on it, it definitely was using the model and would throw an exception if I tried to return the wrong structure.

Reply:
huh, this is interesting -- We probably need to figure out what the best practice here is, and then work on standardizing our results!

Reply:
I already talked about this with Jess, but for anybody else following these comments: the `response_model` does execute on what gets returned from the view function. The downside is that you get screwy output typing if you try to type annotate the view func because of the interaction with the decorator. The unwrapped view function here returns `dict[str, Any]`, but once the router decorator with `response_model` has worked its magic, it's returning a `PangoLineagesResponse`.

It would be possible to drop the `response_model` and instead just do the schema in the view func code
```return PangoLineagesResponse.parse_obj({"lineages": all_lineages})```
or something similar, but the downside to that is FastAPI won't use the response schema in its automatically generated docs.

Another option would be to do the above, but _also_ put in the `response_model`. But -- after a bit of experimentation -- it looks like that just means FastAPI winds up init-ing the output schema twice. Once when we directly use the schema in the view func code, then again when FastAPI is doing the automatic processing via `response_model`. I was hoping it would see that the type matched and just directly use the schema object that was returned, but it seems to run twice.

Finally, I know FastAPI does a lot of implicit stuff based on types (eg, a view func arg like this `view_func(incoming: MyModel)` causes FastAPI to parse the request data view MyModel, then expose that as the arg `incoming`, so I was hoping there might be something similar with doing a type annotation of the output from `list_pango_lineages` in connection to the model, avoiding the double init, but I didn't see anything. As far as I can tell, FastAPI ignores the view func's output type.

Since the automatic API docs are nice and we don't really consume the view func's returned value in Python code, it seems like it's slightly better to use `response_model` and have the view func's return be a dict that matches to what the model uses.